lib/wda/webdriveragent.js

```diff
@@ -231,6 +235,12 @@ class WebDriverAgent {
       }
     }
 
+    if (this.idb && this.isBundledWDAProvided()) {
+      log.info('Will launch WDA with idb instead of xcodebuild since it is provided');
+      await this.prepareWDA();
+      return await this.idb.runXCUITest(WDA_BUNDLE_ID, SAFARI_BUNDLE_ID, WEBDRIVERAGENT_TEST_BUNDLE_ID);
```

Comment:
why safari?

Reply:
No particular reason. I needed to pass something so I pass safari. I can try to pass the WDA id twice